### ISSUE TITLE
Uncaught TypeError: _this.actions.initCoords is not a function

### ISSUE THREAD

**Issue by DirtZERO** [cormacrelf/angular-skyhook#204]
Uncaught TypeError: _this.actions.initCoords is not a function
    at HTML5Backend.handleTopDragStart (HTML5Backend.js:65)
    at ZoneDelegate.push.../node_modules/zone.js/dist/zone.js.ZoneDelegate.invokeTask (zone.js:421)
    at Object.onInvokeTask (zone.js:299)
    at ZoneDelegate.push.../node_modules/zone.js/dist/zone.js.ZoneDelegate.invokeTask (zone.js:420)
    at Zone.push.../node_modules/zone.js/dist/zone.js.Zone.runTask (zone.js:188)
    at ZoneTask.push.../node_modules/zone.js/dist/zone.js.ZoneTask.invokeTask [as invoke] (zone.js:496)
    at invokeTask (zone.js:1540)
    at globalZoneAwareCallback (zone.js:1566)

I apologize if this is not where I should ask this but I'm getting this error run time error when I try to start a drag and I'm not sure what i might have done wrong. Has any one else seen this error? any help or suggestions would be very appreciated.

**Comment by cormacrelf** [comment]
It was added to dnd-core recently, and you’re using a newer major version of the HTML5 backend that requires it. 

https://github.com/react-dnd/react-dnd/pull/1082/files

Temp fix would be to downgrade the HTML5 backend; however I will shortly push a new skyhook core. Did you get any peer dependency warnings when you installed a newer HTML5? Or did they react-dnd team flub their semver? I would doubt that, they do a pretty good job. Maybe I flubbed mine.
